projects/js-packages/publicize-components/src/components/services/custom-inputs.tsx

```diff
@@ -1,8 +1,9 @@
 import { Alert } from '@automattic/jetpack-components';
 import { ExternalLink } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
-import { createInterpolateElement, useId } from '@wordpress/element';
+import { createInterpolateElement, useCallback, useId, useState } from '@wordpress/element';
 import { __, _x } from '@wordpress/i18n';
```

Comment:
You will need this for the other suggestion

```suggestion
import { __, _x, sprintf } from '@wordpress/i18n';
```